master/reference/calicoctl/resources/policy.md

```diff
@@ -65,6 +69,15 @@ the policy is enforced after connection tracking and any DNAT.
 See [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal)
 for how `doNotTrack` and `preDNAT` can be useful for host endpoints.
 
+\* If `types` has no value, Calico defaults as follows.
+
+| Ingress Rules Present | Egress Rules Present | `Types` value       |
```

Comment:
IMO, it looks a bit better if you indent this table, since this table is a note to the main table. You can do that by adding `>` in front of the first `|`